src/AVFoundation/AVTypes.cs

```diff
@@ -266,4 +266,39 @@ public static RectangleF WithAspectRatio (this RectangleF self, SizeF aspectRati
 		}
 	}
 #endif
+
+	[Mac (10, 10), NoiOS, NoWatch, NoTV]
+	[StructLayout (LayoutKind.Sequential)]
+	public struct AVSampleCursorSyncInfo {
+		public bool SampleIsFullSync;
```

Comment:
There's multiple native definitions for `bool` (which can be 1 byte or 4 bytes). Can you add a test that check the type(s) and (all) struct sizes and compare them to values you get from ObjC ?